css/style.css

```diff
@@ -2243,3 +2243,26 @@ dt.d_decl:hover .decl_anchor {
         text-align: right;
     }
 }
+
+/**
+ * Message boxes - different color for each category
```

Comment:
As it turns out even without https://github.com/dlang/dlang.org/pull/1778, the message boxes are universally useful.